src/test/java/br/ufpe/cin/mergers/parameters/WhitespaceIgnorationParameterTest.java

```diff
@@ -0,0 +1,51 @@
+package br.ufpe.cin.mergers.parameters;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import br.ufpe.cin.app.JFSTMerge;
+import br.ufpe.cin.files.FilesManager;
+import br.ufpe.cin.mergers.util.MergeContext;
+
+public class WhitespaceIgnorationParameterTest {
+	
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+		//hidding sysout output
+		@SuppressWarnings("unused")
+		PrintStream originalStream = System.out;
+		PrintStream hideStream    = new PrintStream(new OutputStream(){
+		    public void write(int b) {}
+		});
+		System.setOut(hideStream);
+	}
+	
+	public MergeContext getMergeContext(boolean activation) {
```

Comment:
Só uma questão de estilo, mas eu tornaria esse método privado, e o colocaria no fim.

Privado porque só é utilizado internamente. 
Colocaria no fim pois faz mais sentido quem chama ficar acima de quem é chamado. Sobre isso: https://softwareengineering.stackexchange.com/questions/199311/most-human-friendly-way-to-order-class-method-definitions

(Recomendo a leitura de [Clean Code](https://www.amazon.com/dp/0132350882), é bem interessante. )

Reply:
Feito!
O encapsulamento foi descuido meu mesmo. O restante (incluindo os asserts) eu desconhecia, obrigado!